### DISCUSSION TITLE
WiFi Smart Relay ATMS1601

### DISCUSSION THREAD

**Discussion by KaMiKaZePL** [milo526/homebridge-tuya-web#181] [Q&A] [answered]
All,

Tuya App sees the relay and I can control it perfectly.
In the HB it is not seen at all.
Is there any way to solve the issue?

Thanks for feedback.

**Comment by milo526** [comment, marked as answer]
Please post the result of [the steps under "How to check whether the api this library uses can control your device".](https://github.com/milo526/homebridge-tuya-web#how-to-check-whether-the-api-this-library-uses-can-control-your-device)

Without this it is hard for me to help you with regards to this specific issue.

**Reply by KaMiKaZePL** [reply]
Thanks for reaction.
I run the script and got only others Tuya devices I use.
There is nothing related to the ATMS1601s.
I send support request to Tuya as you have recommended.
I keep you updated.

**Reply by KaMiKaZePL** [reply]
No respond from Tuya support.
The flash seams to be the only option for the device.
